src/mark.ts

```diff
@@ -99,6 +103,14 @@ export interface MarkConfig extends ColorMixins, VgMarkConfig {
    *
    */
   filled?: boolean;
+
+  /**
+   * The tooltip text string to show upon mouse hover or an object defining which fields should the tooltip be derived from.
+   *
+   * - If `tooltip` is `{"fields": "encodings"}`, then all fields in encodings will be used.
+   * - If `tooltip` is `{"fields": "data"}`, then all fields in the underlying data will be used.
```

Comment:
"all fields that appear in the data will be used"

Reply:
Meh.